web/src/validators/isThirdPerson.js

```diff
@@ -0,0 +1,8 @@
+// custom vuelidate validator to ensure input does not contain first person
+const isThirdPerson = (value) => {
+  const firstPersonRegex = /((?<=[\s,.:;"’']|^)I(?=[\s,.:;"’'])|わたし|私)/;
```

Comment:
Adding subject marking participle 'は' and `^`. Thoughts?
```suggestion
  const firstPersonRegex = /((?<=[\s,.:;"’']|^)I(?=[\s,.:;"’'])|^わたしは|^私は)/;
```